### ISSUE TITLE
Add Integration Examples

### ISSUE THREAD

**Issue by pgsmith123** [vmware/terraform-provider-vra#540]
### Code of Conduct

This project has a Code of Conduct that all participants are expected to understand and follow:
* [x] I have read and agree to the project's [Code of Conduct](https://github.com/vmware/terraform-provider-vra/blob/main/CODE-OF-CONDUCT.md)

### Description

It would be extremely helpful to provide examples where arguments are vague like vra_integration.integration_properties and vra_integration.custom_properties. I have tried inputting properties in various ways to create an Active Directory integration but can not get it to work. Ive taken all the properties as defined in the /iaas/api/integrations post method and from importing an existing Active Directory integration into Terraform. I can not get either one to work and the error is not helpful.

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

**Comment by frodenas** [comment]
@pgsmith123 There are a few integration examples at the [examples](https://github.com/vmware/terraform-provider-vra/blob/main/examples/integration/main.tf) directory, including an Active Directory integration. Have you tried that example? is there anything else that you are looking for?

**Comment by pgsmith123** [comment]
@frodenas I did not see that. It looks like what i need. Ill give it a try today. It would be exceptionally cool if that made it into the providers resource page somehow.

**Comment by pgsmith123** [comment]
@frodenas I followed the example, i have everything set except for the endpointId. Is that something that should be populated? If so where would i get that from?

Error: Failed to validate credentials. Error: [ad-integration] Failed to submit sync action [action-id: 8a768ba96b4f9716016b5276c1890058] for execution.  /resources/endpoints/null

The credentials used in the terraform are exactly the same as the ones used in the GUI.

```
resource "vra_integration" "ad" {
  name                   = "ad.domain.com"
  integration_type       = "activedirectory"
  private_key            = data.vault_generic_secret.aria_automation.data["${var.tier}_ad_password"]
  #private_key = "password"

  integration_properties = {
    server: "ldaps://ad.domain.com"
    user: data.vault_generic_secret.aria_automation.data["${var.tier}_ad_username"]
    #user: "username"
    defaultOU: "dc=ad,dc=domain,dc=com"
    connectionTimeout: 10
    
    certificate: <<-EOT
-----BEGIN CERTIFICATE-----
CERT
-----END CERTIFICATE-----
EOT
  }
}

```

**Comment by frodenas** [comment]
Yes, the `endpointId` is a mandatory property, as it should be the `ID` of the runtime environment endpoint. Unfortunately, there is no way to retrieve this `ID` from a data source; you will have to manually get that `ID` by going to the Integrations page, add an AD integration, and inspect the page to see the network calls and get the output of the endpoints API call.
